repos/system_upgrade/el7toel8/libraries/utils.py

```diff
@@ -82,8 +82,7 @@ def call_with_oserror_handled(cmd):
                     'hint': 'Please ensure that {} is installed and executable.'.format(cmd[0])
                 }
             )
-        else:
-            raise StopActorExecutionError(
+        raise StopActorExecutionError(
```

Comment:
wtf?

Reply:
the point here is to avoid these error: `Unnecessary "else" after "raise"'`    there's no need to have these `else`, because everything beside ` if cmd: ` will raise that message... 

or I'm think wrong here?

Reply:
@pirat89 Ok by me

Reply:
You are right. Sorry about the reaction. But looking into the function, the if statement doesn't make sense at all. @vinzenz could you look at the function? It really seems that the `if cmd` in the way it is used here, is wrong. So not sure whether you wanted to check something else or the message in the second raise should be different.